02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>On the <code>logger[('ETA')]</code>  given Python's tuple conventions is the same as <code>logger['ETA']</code>  which works obviously, but if you are trying to teach to by default use tuples then it should be <code>logger[('ETA',)]</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Good catch. I changed it to <code>('User', 'ETA')</code>  to make it clear that this is a tuple.</p>

Reply:
<p>I would add a short comment indicating that <code>'User'</code>  is in no way special, otherwise I could easily see people assuming that <em>all</em> user-defined loggable quantities must be placed in this namespace.</p>

Reply:
done